src/main/java/at/hannibal2/skyhanni/features/nether/kuudra/KuudraAPI.kt

```diff
@@ -26,6 +27,26 @@ object KuudraAPI {
         "§.\\s*(?:§.)*KUUDRA DOWN!"
     )
 
+    private val kuudraTiers = listOf("HOT_", "BURNING_", "FIERY_", "INFERNAL_")
+
+    private val kuudraArmors: List<String>
```

Comment:
instead of this list, please use a regex, like https://github.com/hannibal002/SkyHanni/blob/7efb11313d70c65b8df6692ed237f137d4d346d1/src/main/java/at/hannibal2/skyhanni/features/garden/farming/ArmorDropTracker.kt#L39